### DISCUSSION TITLE
IPTVNator Git channels and EPG

### DISCUSSION THREAD

**Discussion by rfnxinger** [community/community#64428] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi there,

I installed IPTVNator and am very happy with it.
The only feature I'm missing is to show ist the EPG dialogue.

![image](https://github.com/community/community/assets/31359191/9626bbe7-14a1-4864-998f-6f72d8a8a0c8)

I browsed for about 2 hours and already found out that I can add 

`https://github.com/iptv-org/epg/tree/master/sites
`![image](https://github.com/community/community/assets/31359191/1cbb6c92-5943-4805-8bd0-6ab14621bf88)

into the "TV-Program (EPG) URL" Box, what seems to work but I cant find the right shortcut or a  GUI button to make IPTVNator show the EPG !

Could anybody please tell me how to make the EPG show?

Thank you 

rfnX

**Comment by AhmedNasser1601** [comment]
It's great to hear that you're enjoying IPTVNator! It looks like you're trying to set up the Electronic Program Guide (EPG) in the application and are having trouble finding the right way to display it. Here are the steps you can follow to show the EPG in IPTVNator:

1. **Add EPG Source:**
   You've already discovered that you need to add the EPG source URL. You mentioned using the URL `https://github.com/iptv-org/epg/tree/master/sites`, which should work. Make sure you've added it correctly in the "TV-Program (EPG) URL" box.

2. **Refresh EPG Data:**
   After adding the EPG source URL, you'll need to refresh or update the EPG data. There might be a refresh button, a sync option, or the application might automatically update the EPG data after you add the source URL. Look for any option that allows you to initiate the update of EPG data.

3. **Accessing EPG:**
   Once the EPG data is updated, you should be able to access the EPG within the IPTVNator interface. Look for a "TV Guide" or "EPG" button/icon within the application. It might be located in the main menu or somewhere in the settings area.

4. **Navigating EPG:**
   When you access the EPG, you'll likely see a grid displaying various TV channels and their respective schedules. You should be able to scroll through the guide to see what's currently airing and what's scheduled in the future.

5. **EPG Display Modes:**
   Depending on the application's interface, there might be different ways to view the EPG, such as a timeline view, grid view, or list view. Explore these options to find the one that suits your preference.

6. **Keyboard Shortcuts or GUI Buttons:**
   If the application supports keyboard shortcuts, there might be a combination of keys that you can use to directly access the EPG. Alternatively, some applications have GUI buttons like "Guide" or "EPG" that you can click to open the EPG.

Remember, the steps provided here are based on common practices for setting up and accessing EPG data in IPTV applications. Since IPTVNator might have specific features and settings, it's a good idea to refer to the application's documentation or user manual for detailed instructions tailored to its interface.

If you still can't find the right way to display the EPG, consider reaching out to the IPTVNator support community or forums where other users might have encountered and solved similar issues.

**Reply by rfnxinger** [reply]
IPTVNator is great!# :1st_place_medal: 
I just had to laugh when I ordered faster Internet a week ago and the agent asked me if I want add 64 TV channels  for €10 plus  ;)

Pt 1: is OK! the URL was entered correctly (https://github.com/iptv-org/epg/tree/master/sites) but when I press the refresh button:
![image](https://github.com/community/community/assets/31359191/54c679f7-9230-4b57-bf33-ca6798fcaf21)
I get an error bottom right:
![image](https://github.com/community/community/assets/31359191/73450209-1a86-425e-b21d-73b0acb60497)
( Could not load EPG )

Are there other source URLs I can try or could I maybe have another connection issue eg a closed port?

Watching channels was never a problem ...

**Reply by 4gray** [reply]
Hey @rfnxinger. Thank you for the feedback. In iptvnator, a direct link to a `*.xml` or `*.xml.gz` file is expected.

**Reply by rfnxinger** [reply]
thank you @4gray !

Nice to know that IPTVnator expects a XML file.
But the  correct XML file related to  https://github.com/iptv-org/epg/tree/master/sites is hard to find for me.

Also searched https://github.com/iptv-org/awesome-iptv#epg-sources but couldnt find the link to the correct XML file.

Would you please do me the favour and get me the link or give me a hint how me can find it?

Here a screenshot of all m3u files installed:

![image](https://github.com/community/community/assets/31359191/b642a210-6025-48c8-b45a-599d022f20cc)

while I do not really care of the EPG for the xxx.m3u ;)

found the  m3u I use most times here: 

https://iptv-org.github.io/iptv/languages/deu.m3u 

but I cant find the related xml file


rfnX

**Reply by rfnxinger** [reply]
OK!

I will need to find a m3u in the git network that has the related xml file appended.


Thank you all!

Thread closed!

rfnX

**Comment by rfnxinger** [comment]
OK no matter!
Dont want to bug y'all

I'll try to find a m3u that contains all channels I need and also has the related xml file for download.

Thread can be closed.

ThX for your help

rfnX
